wire-ios/Wire-iOS/Sources/Helpers/ColorScheme.swift

```diff
@@ -20,12 +20,19 @@ import Foundation
 import UIKit
 import WireUtilities
 import WireCommonComponents
+import SwiftUI
 
 @objc
 enum ColorSchemeVariant: UInt {
     case light, dark
 }
 
+extension Color {
+    static var backgroundColor = Color(red: 0.93, green: 0.94, blue: 0.94)
+    static var customRed = Color(red: 0.76, green: 0, blue: 0.07)
+    static var customGreen = Color(red: 0.11, green: 0.47, blue: 0.2)
+}
```

Comment:
I noticed we have Colors.xcassets for our project-wide color scheme and SemanticColors.swift for semantic color definitions in light and dark mode. Could you help me understand the reasoning for not following the same pattern in this PR? I think it would be beneficial to maintain consistency and avoid defining colors in multiple places. I'm open to discussion on this if there are specific reasons for a different approach here.

Reply:
I agree. I will move them to Color assets